dev-tools/reconfigurator-exec-unsafe/src/main.rs

```diff
@@ -245,7 +245,7 @@ impl ReconfiguratorExec {
                 blueprint: &blueprint,
                 creator: OmicronZoneUuid::from_untyped_uuid(creator),
                 sender,
-                mgs_updates,
+                mgs_updates: mgs_updates.clone(),
```

Comment:
This was needed because the driver shuts down when the sender side of its watch channel closes.  We don't want that to happen until later, now at L299 (in the new code).

Reply:
could you add a comment here and in `drop` below?

Reply:
Done.